src/features/bookOffer/components/BookingInformations.native.test.tsx

```diff
@@ -127,6 +130,31 @@ describe('<BookingInformations />', () => {
     expect(screen.getByTestId('price-line__label')).toBeTruthy()
   })
 
+  it('should display stock attributes', () => {
```

Comment:
Tu peux ajouter au moins 1 test supplémentaire je pense :
- should not display stock attributes when feature flag deactivated (et que tu en as)
A voir ce que tu en penses mais potentiellement moins important :
- should not display stock attributes when feature flag activated (et que tu n'en as pas, genre t'as pas ton tiret)

Reply:
Pour être plus précis tu peux faire 
```
describe('When wipAttributesCinemaOffers feature flag activated'){
it('should display stock attributes when the offer has it')
it('should not display stock attributes when the offer has not it')
}

describe('When wipAttributesCinemaOffers feature flag deactivated'){
it('should not display stock attributes when the offer has it')
it('should not display stock attributes when the offer has not it')
```